batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/constants/UrlConstants.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.batch7crud.constants;
+
+import java.io.File;
+
+/**
+ * Created by Romit Amgai <romitamgai@lftechnology.com> on 1/21/16.
+ */
+public class UrlConstants {
+  public static final String URL_SEPARATOR = File.separator;
+  public static final String WEB_INF = "WEB-INF";
+  public static final String VIEWS = "views";
+
+  public static final String EMPLOYEE_ROUTE = URL_SEPARATOR + "employees";
+  public static final String EMPLOYEE_LIST_PAGE = URL_SEPARATOR + WEB_INF + URL_SEPARATOR + VIEWS + URL_SEPARATOR + "employeesList.jsp";
```

Comment:
man are you serious??
why not make VIEWS = sep + webinf + views + sep; and use it for the sake of simplicity